ingest/bin/transform-authors

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
```

Comment:
*(minor suggestion)*

use `ingest/transform:` in commit title for consistency

Reply:
❤️ consistency. Reworded in ff61804c815c5cddc19360e41eeb4e60dfffe21f